app/styles/theme.scss

```diff
@@ -1,3 +1,4 @@
+$Theme-margin-xSmall: 13px;
```

Comment:
Is this value used anywhere else in the project? Seems an oddly specific number.